Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1

```diff
@@ -82,7 +88,12 @@ $ErrorActionPreference = 'Stop'
 $msDeployExePath = Get-MsDeployExePath
 
 # Ensure that at most a package (.zip) file is found
-$packageFile = Get-SinglePackageFile -package $Package
+$packageFile = Get-SingleFilePath -file $Package
```

Comment:
This can be renamed to $packageFilePath to be consistent with the newly added variable